tests/kuttl/tests/fernet_rotation/01-assert.yaml

```diff
@@ -0,0 +1,5 @@
+# create a test token, save it in /tmp/
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: $KEYSTONE_KUTTL_DIR/../common/scripts/create_test_token.sh
```

Comment:
starting with kuttl 0.17.0 should be relative to the test dir, so `../../common/scripts/validate_test_token.sh`. otherwise you could not run it manually from the base dir